packages/app/src/components/VisualizationTypeSelector/VisualizationTypeSelector.js

```diff
@@ -57,7 +57,7 @@ export const VisualizationTypeSelector = (
     const getVisTypes = () => Object.keys(visTypeDisplayNames)
 
     const VisTypesList = (
-        <Card dataTest={'visualization-type-selector-card'}>
+        <div dataTest={'visualization-type-selector-card'}>
```

Comment:
I think this is the source of the failing Cypress tests. Custom components receive `dataTest={'name'}` while regular html elements use `data-test='name'`. e.g. `data-test={'visualization-type-selector-button'}` on the `div` further down in the same file as this.

Reply:
```suggestion
        <div data-test='visualization-type-selector-card'>
```

Reply:
Great spot, thank you @martinkrulltott